app/javascript/products.vue

```diff
@@ -1,74 +1,88 @@
-        v-else-if='product_n_days_passed.elapsed_days === 7'
-      )
-        h2.card-header__title
-          | {{ product_n_days_passed.elapsed_days }}日以上経過
-          span.card-header__count(v-if='selectedTab === "unassigned"')
-            | （{{ countProductsGroupedBy(product_n_days_passed) }}）
-      header.card-header.a-elapsed-days(v-else)
-        h2.card-header__title
-          | {{ product_n_days_passed.elapsed_days }}日経過
-          span.card-header__count(v-if='selectedTab === "unassigned"')
-            | （{{ countProductsGroupedBy(product_n_days_passed) }}）
-      .card-list(:class='listClassName')
-        .card-list__items
-          product(
-            v-for='product in product_n_days_passed.products',
-            :key='product.id',
-            :product='product',
-            :currentUserId='currentUserId',
-            :isMentor='isMentor'
+          //- prettier-ignore: need space between v-if and id
+          header.card-header.a-elapsed-days(
+            v-if='product_n_days_passed.elapsed_days === 0', id='0days-elapsed'
+          )
+            h2.card-header__title
+              | 今日提出
+              span.card-header__count(v-if='selectedTab === "unassigned"')
+                | （{{ countProductsGroupedBy(product_n_days_passed) }}）
+          //- prettier-ignore: need space between v-else-if and id
+          header.card-header.a-elapsed-days.is-reply-warning(
+            v-else-if='product_n_days_passed.elapsed_days === 5', id='5days-elapsed'
+          )
+            h2.card-header__title
+              | {{ product_n_days_passed.elapsed_days }}日経過
+              span.card-header__count(v-if='selectedTab === "unassigned"')
+                | （{{ countProductsGroupedBy(product_n_days_passed) }}）
+          //- prettier-ignore: need space between v-else-if and id
+          header.card-header.a-elapsed-days.is-reply-alert(
+            v-else-if='product_n_days_passed.elapsed_days === 6', id='6days-elapsed'
+          )
+            h2.card-header__title
+              | {{ product_n_days_passed.elapsed_days }}日経過
+              span.card-header__count(v-if='selectedTab === "unassigned"')
+                | （{{ countProductsGroupedBy(product_n_days_passed) }}）
+          //- prettier-ignore: need space between v-else-if and id
+          header.card-header.a-elapsed-days.is-reply-deadline(
+            v-else-if='product_n_days_passed.elapsed_days === 7', id='7days-elapsed'
           )
-  unconfirmed-links-open-button(
-    v-if='isMentor && selectedTab != "all" && !isDashboard',
-    :label='`${unconfirmedLinksName}の提出物を一括で開く`'
-  )
+            h2.card-header__title
+              | {{ product_n_days_passed.elapsed_days }}日以上経過
+              span.card-header__count(v-if='selectedTab === "unassigned"')
+                | （{{ countProductsGroupedBy(product_n_days_passed) }}）
+          header.card-header.a-elapsed-days(
+            v-else,
+            :id='[elapsedDaysId(product_n_days_passed.elapsed_days)]'
```

Comment:
入ってくるidは1つなので、[]は必要無さそうです。

```suggestion
            :id='elapsedDaysId(product_n_days_passed.elapsed_days)'
```

Reply:
おっしゃる通りでした！
修正いたしました🙏